packages/react/src/wallet/wallets/safeWallet.ts

```diff
@@ -0,0 +1,10 @@
+import { SafeWallet } from "@thirdweb-dev/wallets";
```

Comment:
SafeWallet should not be exported from root of wallets since it *should* require optional peer dependencies